### PR TITLE
zh-translation:content/zh/news/security/istio-security-2019-005/index.md

### PR DESCRIPTION
**Pull request by kylesliu**
Ref: https://github.com/servicemesher/istio-official-translation/issues/1217

### CODE REVIEW

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 1]
```diff
@@ -16,18 +16,18 @@ aliases:
...
-* For Istio 1.2.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.2.7](/zh/news/releases/1.2.x/announcing-1.2.7) or later.
-* For Istio 1.3.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.3.2](/zh/news/releases/1.3.x/announcing-1.3.2) or later.
+* 对于 Istio 1.1.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.1.16](/zh/news/releases/1.1.x/announcing-1.1.16)。
+* 对于 Istio 1.2.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.2.7](/zh/releases/1.2.x/announcing-1.2.7)。
+* 对于 Istio 1.3.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.3.2](/news/releases/1.3.x/announcing-1.3.2)。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: /news/releases/1.3.x/announcing-1.3.2 -> /zh/news/releases/1.3.x/announcing-1.3.2

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 2]
```diff
@@ -16,18 +16,18 @@ aliases:
...
-* For Istio 1.1.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.1.16](/zh/news/releases/1.1.x/announcing-1.1.16) or later.
-* For Istio 1.2.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.2.7](/zh/news/releases/1.2.x/announcing-1.2.7) or later.
-* For Istio 1.3.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.3.2](/zh/news/releases/1.3.x/announcing-1.3.2) or later.
+* 对于 Istio 1.1.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.1.16](/zh/news/releases/1.1.x/announcing-1.1.16)。
+* 对于 Istio 1.2.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.2.7](/zh/releases/1.2.x/announcing-1.2.7)。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: /zh/releases/1.2.x/announcing-1.2.7 -> /zh/news/releases/1.2.x/announcing-1.2.7

**Comment by tanjunchen** [comment]
@kylesliu First Make Sure CI happy

**Comment by kylesliu** [comment]
> @kylesliu First Make Sure CI happy

The path problem has been modified, but the file [/public/blog/2019/announcing-istio-client-go/index.html] seem to be missing

**Comment by tanjunchen** [comment]
CI happy,content need to  review

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: ISTIO-SECURITY-2019-005
-subtitle: Security Bulletin
-description: Security vulnerability disclosure for CVE-2019-15226.
+title: 安全更新：2019-005
```
Comment (rootsongjc) [inline content/zh/news/security/istio-security-2019-005/index.md]: 保留英文标题

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
...
+subtitle: 安全公告
+description: CVE-2019-15226 的安全漏洞披露。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: CVE-2019-15226 所披露的安全漏洞。

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 5]
```diff
@@ -16,18 +16,18 @@ aliases:
...
+## 内容{#context}
 
-Envoy, and subsequently Istio, are vulnerable to the following DoS attack. Upon receiving each incoming request, Envoy will iterate over the request headers to verify that the total size of the headers stays below a maximum limit. A remote attacker may craft a request that stays below the maximum request header size but consists of many thousands of small headers to consume CPU and result in a denial-of-service attack.
+Envoy 和 Istio 容易受到以下 DoS 攻击。 收到每个传入的请求后, Envoy 将遍历请求标头，以保证请求头的总大小保持在最大限制以下。远程攻击者可能会制作一个请求，该请求的 header 的大小不会超过最大限制，但包含成千上万个小的 header ，来消耗 CPU 并导致拒绝服务攻击。
 
-## Impact and detection
+## 影响范围{#impact-and-detection}
 
-Both Istio gateways and sidecars are vulnerable to this issue. If you are running one of the affected releases, your cluster is vulnerable.
+Istio gateways 和 Sidecars 都容易受到此问题的影响。如果运行的是受影响的发行版之一，则您的群集容易受到攻击。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: Istio gateway 和 sidecar 都容易受到此问题的影响。如果您运行的 Istio 是受影响的发行版本，那么您的集群容易受到攻击。

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 6]
```diff
@@ -16,18 +16,18 @@ aliases:
...
 
-## Mitigation
+## 防范{#mitigation}
 
-* For Istio 1.1.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.1.16](/zh/news/releases/1.1.x/announcing-1.1.16) or later.
-* For Istio 1.2.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.2.7](/zh/news/releases/1.2.x/announcing-1.2.7) or later.
-* For Istio 1.3.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.3.2](/zh/news/releases/1.3.x/announcing-1.3.2) or later.
+* 对于 Istio 1.1.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.1.16](/zh/news/releases/1.1.x/announcing-1.1.16)。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: Pilot、Mixer、Citadel 和 Galley

Comment (tanjunchen) [reply content/zh/news/security/istio-security-2019-005/index.md]: 然后[更新数据平面]

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 7]
```diff
@@ -16,18 +16,18 @@ aliases:
...
+* 对于 Istio 1.2.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.2.7](/zh/news/releases/1.2.x/announcing-1.2.7)。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: ditto

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 8]
```diff
@@ -16,18 +16,18 @@ aliases:
...
+* 对于 Istio 1.3.x 部署: 更新所有控制平面组件 (Pilot, Mixer, Citadel, and Galley) 然后 [更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) 的版本不低于 [Istio 1.3.2]((/zh/news/releases/1.3.x/announcing-1.3.2)。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: ditto

File: content/zh/news/security/istio-security-2019-005/index.md (hunk 6 above)
Comment (kylesliu) [reply content/zh/news/security/istio-security-2019-005/index.md]: 已经修改，请检查

File: content/zh/news/security/istio-security-2019-005/index.md [hunk 9]
```diff
@@ -16,18 +16,18 @@ aliases:
...
+Istio gateway 和 sidecar 都容易受到此问题的影响。如果您运行的 Istio 是受影响的发行版本，那么您的集群容易受到攻击。
 
-## Mitigation
+## 防范{#mitigation}
 
-* For Istio 1.1.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.1.16](/zh/news/releases/1.1.x/announcing-1.1.16) or later.
-* For Istio 1.2.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.2.7](/zh/news/releases/1.2.x/announcing-1.2.7) or later.
-* For Istio 1.3.x deployments: update all control plane components (Pilot, Mixer, Citadel, and Galley) and then [upgrade the data plane](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade) to [Istio 1.3.2](/zh/news/releases/1.3.x/announcing-1.3.2) or later.
+* 对于 Istio 1.1.x 部署: 更新所有控制平面组件(Pilot, Mixer, Citadel, 和 Galley)然后[更新数据平面](/zh/docs/setup/upgrade/cni-helm-upgrade/#sidecar-upgrade)的版本不低于[Istio 1.1.16](/zh/news/releases/1.1.x/announcing-1.1.16)。
```
Comment (tanjunchen) [inline content/zh/news/security/istio-security-2019-005/index.md]: (Pilot, Mixer, Citadel, 和 Galley)->(Pilot、Mixer、Citadel 和 Galley)  一般用中文 、分隔 下面一样

Comment (kylesliu) [reply content/zh/news/security/istio-security-2019-005/index.md]: 已经修改

**Comment by tanjunchen** [comment]
@rootsongjc  ok

**Comment by tanjunchen** [comment]
@rootsongjc need approve

File: content/zh/news/security/istio-security-2019-005/index.md (hunk 9 above)
Comment (kylesliu) [reply content/zh/news/security/istio-security-2019-005/index.md]: 感谢大佬的review

**Pull request merged**